assembly/utils/Bytes.ts

```diff
@@ -28,22 +28,51 @@ export class Bytes {
         return result;
     }
 
-    static copyFromPosition (src: u8[], dst: Array<u8>, position: i32 = 0): void {
+    /**
+    * @description Copy src elements in dst at provided position. 
+    */
+    static copy<T> (src: T[], dst: Array<T>, start: i32 = 0): void {
         for (let i = 0; i < dst.length; i++) {
             if (src.length <= i) {
                 break;
             }
-            dst[i] = src[position + i];
+            dst[start + i] = src[i];
         }
     }
 
-    static copyToPosition (src: u8[], dst: Array<u8>, position: i32 = 0): void {
-        for (let i = 0; i < dst.length; i++) {
-            if (src.length <= i) {
-                break;
-            }
-            dst[position + i] = src[i];
+    static encodeInteger (bytesBuffer: u8[], i: i32): i32 {
+        if (i < 1 << 6) {
+            Bytes.putUint<u8>(bytesBuffer, u8(i) << 2, BIT_LENGTH.INT_8);
+            return 1
```

Comment:
instead of returning 1, can we return `BIT_LENGTH.INT_8`? the same way for `INT_16`, `INT_32`